lib/reset/resetTracker.ts

```diff
@@ -0,0 +1,23 @@
+import dedent from 'dedent';
+
+let resetImported = false;
+
+export const markResetImported = () => {
+  resetImported = true;
+};
+
+export const ensureResetImported = () => {
```

Comment:
Should we put these function definitions behind a `process.env` check so the code isn't included in the production bundle?

Reply:
The other `process.env` checks allow for this entire file to be stripped in prod so no need.

Reply:
Awesome, even better 👍